mixed-format/trino/src/test/java/org/apache/amoro/trino/mixed/ArcticQueryRunner.java

```diff
@@ -16,16 +16,16 @@
  * limitations under the License.
  */
 
-package com.netease.arctic.trino.arctic;
+package org.apache.amoro.trino.mixed;
```

Comment:
The package name confuses me. This package is test for mixed hive format?

Reply:
Not just mixed-hive format, including both mixed-iceberg and mixed-hive.